### PR TITLE
Refresh pvp talents with spell ids

### PR DESCRIPTION
**Pull request by prsauer**

### CODE REVIEW

File: packages/cloud/src/refreshSpellIconsHandler.ts [hunk 1]
```diff
@@ -18,14 +18,18 @@ const gcpCredentials =
 const firestore = new Firestore({
   ignoreUndefinedProperties: true,
   credentials: gcpCredentials,
+  projectId: 'wowarenalogs',
```
Comment (prsauer) [inline packages/cloud/src/refreshSpellIconsHandler.ts:21]: Without this these libraries will check the local env for a gcp project id and use that instead which can lead to some very confusing errors

File: packages/cloud/src/refreshSpellIconsHandler.ts [hunk 2]
```diff
@@ -18,14 +18,18 @@ const gcpCredentials =
...
 const storage = new GoogleCloudStorage({
   credentials: gcpCredentials,
+  projectId: 'wowarenalogs',
 });
+
 const bucket = storage.bucket('images.wowarenalogs.com');
 
 const MATCH_STUBS_COLLECTION = 'match-stubs-prod';
-const NUMBER_OF_MATCHES = 75;
+const NUMBER_OF_MATCHES = 25;
```
Comment (prsauer) [inline packages/cloud/src/refreshSpellIconsHandler.ts:32]: still seeing some OOMs running this; it's parsing shuffles fairly regularly so those are x6 for cpu/mem on parsing

Comment (atinylittleshell) [reply packages/cloud/src/refreshSpellIconsHandler.ts:32]: what does this number control?

**Review by atinylittleshell — APPROVED** [review]

File: packages/cloud/src/refreshSpellIconsHandler.ts (hunk 2 above)
Comment (prsauer) [reply packages/cloud/src/refreshSpellIconsHandler.ts:32]: the number of matches queried per run of the function

**Pull request merged**